Directory.Build.props

```diff
@@ -1,5 +1,5 @@
 <Project>
 	<PropertyGroup>
-        <Version>7.0.0</Version>
+        <Version>7.1.0</Version>
```

Comment:
7.0.0 isn't released yet so you don't need to increment it

Reply:
file update removed now